iOS/Flows/Conversation/Cells/ConversationMessageCell.swift

```diff
@@ -108,15 +105,25 @@ class ConversationMessageCell: UICollectionViewCell {
 
         var snapshot = self.dataSource.snapshot()
 
-        snapshot.deleteItems(snapshot.itemIdentifiers(inSection: .otherMessages))
-        snapshot.appendItems(otherMessages, toSection: .otherMessages)
+        // Clear out the sections to make way for a fresh set of message.
+        snapshot.deleteSections(ConversationMessageSection.allCases)
+        snapshot.appendSections(ConversationMessageSection.allCases)
 
-        snapshot.deleteItems(snapshot.itemIdentifiers(inSection: .currentUserMessages))
+        snapshot.appendItems(otherMessages, toSection: .otherMessages)
```

Comment:
The message cell collection view doesn't animate inserts any more, so the replies should always appear immediately.